config/initializers/cycle_timetables.rb

```diff
@@ -53,7 +38,24 @@
     find_closes: Time.zone.local(2026, 9, 30, 23, 59, 59), # CONFIRMED
     holidays: {
       christmas: Date.new(2026, 12, 18)..Date.new(2026, 1, 5), # TBD
-      easter: Date.new(2026, 3, 18)..Date.new(2026, 4, 1), # TBD
+      easter: Date.new(2026, 3, 23)..Date.new(2026, 4, 6), # TBD
+    },
+  },
+
+}.freeze
+
+FUTURE_TENTATIVE_DATES = {
+  2027 => {
+    find_opens: Time.zone.local(2026, 10, 1, 9), # TBD
```

Comment:
Do we want to add the notes on here for future us? E.g `# First Tuesday of October`

Reply:
Nah, It's elsewhere if we need it. The 2027 dates are definitely just placeholders -- the first tuesday of October in 2027 is actually 6 October.